packages/paste-core/layout/grid/src/types.ts

```diff
@@ -1,12 +1,24 @@
-import {ResponsiveValue} from '@twilio-paste/styling-library';
-import {LayoutProps, FlexboxProps, PaddingProps, Space, SpaceProps} from '@twilio-paste/style-props';
+import type {ResponsiveValue} from '@twilio-paste/styling-library';
+import type {
+  LayoutProps,
+  FlexboxProps,
+  PaddingProps,
+  Space,
+  Margin,
+  Padding,
+  SpaceProps,
+} from '@twilio-paste/style-props';
 
 export interface GridProps extends SpaceProps {
   as?: keyof JSX.IntrinsicElements;
   children: NonNullable<React.ReactNode>;
   equalColumnHeights?: boolean;
   gutter?: Space;
   vertical?: ResponsiveValue<boolean>;
+  marginTop?: Margin;
+  marginBottom?: Margin;
+  marginRight?: Margin;
+  marginLeft?: Margin;
```

Comment:
We're adding margin props to the Grid API?

Reply:
Mmm let me look at this again. I know that previously `marginTop` and `marginBottom` were allowed and applied to the grid stylings if provided (regardless of whether it is a vertical grid or not).

Reply:
To follow up, _technically_ these margin props were already available on the Grid API because the props interface extends `SpaceProps`, [which has them available](https://github.com/twilio-labs/paste/blob/8b3924075e2ea5b890f5c0a90c3fc9985ec38708/packages/paste-style-props/src/types/space.ts#L32).